### ISSUE TITLE
Create an example that uses the .belowIfTooWide modifier

### ISSUE THREAD

**Issue by jordibruin** [jordibruin/Dynamic-Islands#3]
I have no clue how to use this. Here's what Apple says about it:

If content is too wide to appear in a leading position next to the TrueDepth camera, use the [belowIfTooWide](https://developer.apple.com/documentation/WidgetKit/DynamicIslandExpandedRegionVerticalPlacement/belowIfTooWide) modifier to render leading content below the TrueDepth camera.

https://developer.apple.com/documentation/activitykit/displaying-live-data-with-live-activities

**Comment by shamanskyh** [comment]
I think you can use this in combination with the `dynamicIsland(verticalPlacement:` modifier:
```
.dynamicIsland(verticalPlacement: .belowIfTooWide)
```

**Comment by jordibruin** [comment]
@shamanskyh it doesn't change anything for me though. Does it work for you?

**Comment by shamanskyh** [comment]
Nope, I also have trouble figuring out what, if anything, about my layout changes when that's used.
